x-pack/plugins/reporting/server/export_types/common/get_absolute_url.ts

```diff
@@ -17,10 +17,10 @@ export const getAbsoluteUrlFactory = ({
   protocol,
   hostname,
   port,
-  defaultBasePath,
+  defaultBasePath, // never gets an override
```

Comment:
Does this mean we should remove this? Sorry not clear on what this is doing since it sounds like it's mostly unused?

Reply:
It's pre-existing code, but is related to this PR so it make sense to remove it.